src/main/java/edu/osu/waiting4ubackend/client/DBClient_user.java

```diff
@@ -0,0 +1,60 @@
+package edu.osu.waiting4ubackend.client;
```

Comment:
We only need one DBClient class that means all the queries interact with the database have to write into one file. Could you merge these codes into DBClient.java?